src/common/string/has-template.ts

```diff
@@ -1,4 +1,5 @@
-const isTemplateRegex = new RegExp("{%|{{|{#");
+const isTemplateRegex = new RegExp("{%|{{");
```

Comment:
Removing the comment. Although it's template syntax, it doesn't impact the output.